packages/molenc/molenc.5.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/molenc"
+bug-reports: "https://github.com/UnixJunkie/molenc/issues"
+dev-repo: "git+https://github.com/UnixJunkie/molenc.git"
+license: "BSD-3"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: [
+  # ["cp" "bin/molenc_ph4_type_atoms.py" "%{bin}%/molenc_ph4_type_atoms.py"]
+  ["cp" "bin/molenc_common.py" "%{bin}%/molenc_common.py"]
+  ["cp" "bin/molenc_type_atoms.py" "%{bin}%/molenc_type_atoms.py"]
+  ["cp" "bin/molenc_scan.py" "%{bin}%/molenc_scan.py"]
+  ["cp" "bin/molenc.sh" "%{bin}%/molenc.sh"]
+  ["cp" "bin/molenc_scan.sh" "%{bin}%/molenc_scan.sh"]
+]
+depends: [
+  "bst"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Oops I was looking at the wrong log, discard my previous (deleted) comment.
Anyway, the `dune-project` file in the repository says this package requires at least dune 1.11